main.js

```diff
@@ -162,10 +162,134 @@ const inventory = [
     },
 ];
 
-function calculateAmountToSell(array) {
+
+function calculateAmountToSell(inventoryArray) {
     let stockToSell = 0;
     for (const inventoryKey in inventory) {
-        stockToSell += inventoryKey.originalStock - inventoryKey.sold;
+        stockToSell += inventory[inventoryKey].originalStock - inventory[inventoryKey].sold;
     }
     return stockToSell;
-}
\ No newline at end of file
+}
+
+function calculateRevenueTarget(inventoryArray) {
+    let revenue = 0;
+    for (const inventoryKey in inventoryArray) {
+        revenue += (inventoryArray[inventoryKey].originalStock - inventoryArray[inventoryKey].sold) * inventoryArray[inventoryKey].price;
```

Comment:
De target is de uiteindelijke opbrengst. Je kunt hier dus de `.sold` weglaten. Ik snap de verwarring. Je berekend nu in feite de opbrengst die je bovenop de huidige opbrengst nog gaat krijgen, maar we zijn geïnteresseerd in de uiteindelijk totale opbrengst.